test/e2e/pages/self-serve-page.ts

```diff
@@ -80,26 +82,35 @@ export class SelfServePage {
   }
 
   async checkIMAPInfo(expectedInfo: connectionInfo) {
-    expect(await this.imapServerName.innerText()).toBe(`Server Name: ${expectedInfo['serverName']}`);
-    expect(await this.imapServerPort.innerText()).toBe(`Server Port: ${expectedInfo['serverPort']}`);
+    expect(await this.imapServerName.innerText({ timeout: 5000 })).toBe(`Server Name: ${expectedInfo['hostName']}`);
+    expect(await this.imapServerPort.innerText()).toBe(`Server Port: ${expectedInfo['port']}`);
     expect(await this.imapSecurityType.innerText()).toBe(`Security: ${expectedInfo['securityType']}`);
-    expect(await this.imapUsername.innerText()).toBe(`Username: ${expectedInfo['userName']}`);
+    // when running on the stage env a thundermail email is required, but might not exist on the local dev env
+    if (ACCTS_TARGET_ENV == 'stage') {
+      expect(await this.imapUsername.innerText()).toBe(`Username: ${expectedInfo['userName']}`);
+    }
     expect(await this.imapPassword.innerText()).toBe(`Password: ${expectedInfo['appPassword']}`);
   }
 
   async checkJMAPInfo(expectedInfo: connectionInfo) {
-    expect(await this.jmapServerName.innerText()).toBe(`Server Name: ${expectedInfo['serverName']}`);
-    expect(await this.jmapServerPort.innerText()).toBe(`Server Port: ${expectedInfo['serverPort']}`);
+    expect(await this.jmapServerName.innerText({ timeout: 5000 })).toBe(`Server Name: ${expectedInfo['hostName']}`);
+    expect(await this.jmapServerPort.innerText()).toBe(`Server Port: ${expectedInfo['port']}`);
     expect(await this.jmapSecurityType.innerText()).toBe(`Security: ${expectedInfo['securityType']}`);
-    expect(await this.jmapUsername.innerText()).toBe(`Username: ${expectedInfo['userName']}`);
+    // when running on the stage env a thundermail email is required, but might not exist on the local dev env
+    if (ACCTS_TARGET_ENV == 'stage') {
+      expect(await this.jmapUsername.innerText()).toBe(`Username: ${expectedInfo['userName']}`);
+    }
```

Comment:
I think it's fair to just expect username, as they'll need to login even see that page properly.

Reply:
Sorry not understanding your comment, the test does login first when running locally or on the local dev env (when running on the local dev env it signs in using stage FxA but goes back to localhost for the self-serve page).

Reply:
Ah yeah, I think I've looked at too many PRs. 🫠 Thanks for the info!